admin/worker/deployments_health_check.go

```diff
@@ -88,21 +87,46 @@ func (w *Worker) deploymentHealthCheck(ctx context.Context, d *database.Deployme
 		return nil
 	}
 
-	if !isRuntimeHealthy(resp) {
-		s, _ := protojson.Marshal(resp)
-		w.logger.Error("deploymentsHealthCheck: runtime is unhealthy", zap.String("host", d.RuntimeHost), zap.ByteString("health_response", s))
+	if runtimeUnhealthy(resp) {
+		f := []zap.Field{zap.String("host", d.RuntimeHost)}
+		if resp.LimiterError != "" {
+			f = append(f, zap.String("limiter_error", resp.LimiterError))
+		}
+		if resp.ConnCacheError != "" {
+			f = append(f, zap.String("conn_cache_error", resp.ConnCacheError))
+		}
+		if resp.MetastoreError != "" {
+			f = append(f, zap.String("metastore_error", resp.MetastoreError))
+		}
+		if resp.NetworkError != "" {
+			f = append(f, zap.String("network_error", resp.NetworkError))
+		}
+		w.logger.Error("deploymentsHealthCheck: runtime is unhealthy", f...)
```

Comment:
nit: can we put `deployment health check: ...` instead of `deploymentsHealthCheck` in the logs? The other jobs do that